### ISSUE TITLE
[a11y-defect-FSR]: Signature Field doesn't announce properly on Statement of Truth component 

### ISSUE THREAD

**Issue by josephrlee** [department-of-veterans-affairs/va.gov-team#66853]
### Point of contact

Joseph Lee

### Severity level

4, Minor. Consider fixing or exploring in 2-4 sprints post-launch.

### Details

On the Statement of Truth component in the Review & Submit page, the full name error message needs to be put in an aria-live region and be made polite so that it reads the error message better. (suggestion from Angela Fowler).

From Ticket #71039 (duplicate):
A session with a JAWS user during the accessibility study of the streamlined waiver path of eFSR hosted in November 2023 revealed that the signature field is not announced on JAWS. 

Scenario: JAWS does not announce that the user is filling out a signature to allow them to submit the form

Given: JAWS does not announce that the user is filling out a signature before submitting the form / Veteran does not know to sign or where to sign the form
When: Navigates to the signature field
Then: Might not be able to sign and won't be able to submit the form

### Reproduction steps

1. Fill out FSR: https://staging.va.gov/manage-va-debt/request-debt-help-form-5655/
2. Navigate to review & submit page
3. Fill full name out incorrectly to get error to trigger
...

### Proposed solution or next steps

Add an aria-live region and make it polite. 

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Axe-core
- [X] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by denisecoveyduc** [comment]
@Tom-Davis Shifting this one to a high-priority as there are some legal implications here.

**Comment by Tom-Davis** [comment]
@denisecoveyduc is this high enough priority to swap out for current sprint work now or just make it #1 for the next sprint?

@HeatherRienks FYI

**Comment by HeatherRienks** [comment]
Per @denisecoveyduc , we definitely need to get this done, but can push to Sprint 122 if needed.

@Tom-Davis

**Comment by josephrlee** [comment]
@Dr-Pongo, here's the guidance from the DS: https://design.va.gov/components/form/statement-of-truth

They have a custom web component for the Statement of Truth input field that's currently [custom on our version](https://github.com/department-of-veterans-affairs/vets-website/blob/69e85955e6ebb113ac6cf32349d81c38f4658a9e/src/applications/financial-status-report/components/shared/PreSubmitSignature.jsx).  I'm guessing adopting the DS one should fix the screenreader issue.

**Comment by josephrlee** [comment]
Here's a rough example of how it can implemented:

```
<div
  style={{
    maxWidth: 600
  }}
>
  <VaStatementOfTruth
    checkboxError=""
    checkboxLabel="I certify the information above is correct and true to the best of my knowledge and belief."
    heading="This is a custom heading"
    inputError=""
    inputLabel="Your full name"
    inputMessageAriaDescribedby=""
    inputValue=""
    onVaCheckboxChange={function noRefCheck(){}}
    onVaInputBlur={function noRefCheck(){}}
    onVaInputChange={function noRefCheck(){}}
  >
        <p>
          I've reviewed the information I provided in this request, including:
        </p>
        <ul>{renderStatementOfTruthItems()}</ul>
  </VaStatementOfTruth>
</div>
```

**Comment by OptionSelect** [comment]
Per discussion [here](https://dsva.slack.com/archives/G01KBHYEXV2/p1711566305303359) moving forward with initial ticket attempting to get current iteration to announce properly. Later ticketed work will be for implementing the new component once it is in a more complete state.

**Comment by OptionSelect** [comment]
Based on convo with Andrew and Aaron we found that the aria message for the VaTextInput component _does_ exist; however, its at a level lower than the surface and must be accessed by the respective screenreader's more context keybind. It would seem I need to speak with CAIA (reached out to Megan for how to contact) and/or Platform to get this message raised to a different level. OR we would need another interactive element on the screen to read this out rather than the signature field.

[Uploading... image.png]

**Comment by OptionSelect** [comment]
Reached out to accessibility team [here](https://dsva.slack.com/archives/C8E985R32/p1712003691475189)

**Comment by OptionSelect** [comment]
Per convo with Accessibility team and Megan in above thread link, Megan is creating another design system ticket requesting the desired behavior for Platform to implement.

Dan linked a couple things where this has been brought to attention in the past and I guess is currently being discussed
[not being announced by screenreaders on mac](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2572)
[alternatives being researched])https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2619) that seem to corroborate my experience in testing this one locally

**Comment by OptionSelect** [comment]
Design System Ticket [here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2649)
